modules/components/RouteHandler.js

```diff
@@ -12,7 +12,7 @@ class RouteHandler extends React.Component {
 
   getChildContext() {
     return {
-      routeHandlers: this.context.routeHandlers.concat([ this ])
+      depth: this.context.depth + 1
```

Comment:
Can we please call this `routeDepth`, just so we know it has something to do with the router?